q2_taxa/plugin_setup.py

```diff
@@ -60,6 +60,116 @@
                 'all features will be summed when they are collapsed.'
 )
 
+plugin.methods.register_function(
+    function=filter_table,
+    inputs={
+        'taxonomy': FeatureData[Taxonomy],
+        'table': FeatureTable[Frequency]
+    },
+    parameters={'include': qiime2.plugin.Str,
+                'exclude': qiime2.plugin.Str,
+                'mode':
+                    qiime2.plugin.Str % qiime2.plugin.Choices(['exact',
+                                                               'contains']),
+                'query_delimiter': qiime2.plugin.Str},
+    outputs=[('filtered_table', FeatureTable[Frequency])],
```

Comment:
I guess it wouldn't be a great idea to return a table that contained the removed features as that could be empty.

Reply:
Yeah, and it's also not something that is very useful in the most common use cases (i.e., removing mitochondria or chloroplast sequences).